### ISSUE TITLE
Saving house styles to Mapzen breaks relative urls to resources (e.g. fonts, textures)

### ISSUE THREAD

**Issue by louh** [tangrams/tangram-play#660]

**Comment by louh** [comment]
Given the conversations I've had so far, it seems to the best course of action is to not let this affect launch. @souperneon @kkowalsky @nvkelso @bcamper

**Comment by nvkelso** [comment]
All Mapzen basemap styles are now provided (since last week) in the ZIP bundle format via `mapzen.com/carto`. Is there a problem with Tangram Play referencing those ZIP bundles instead of the YAML?

I'm more in favor of "importing" the Mapzen basemap styles versus "opening" the full source by default. I suspect more people will want to overlay their own data on top of our basemap styles, or do a tiny modification of a basemap style (e.g. change labels language, enable transit), than go thru 6,000 lines of basemap source code.

**Comment by louh** [comment]
Yeah - we can definitely revisit this after release. I talk a little more about my rationale here: https://github.com/tangrams/tangram-play/issues/640#issuecomment-275501693
